app/controllers/home_controller.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 class HomeController < ApplicationController
-  # def index
-  # end
+  def index
+    @projects = Project.last(9).reverse
```

Comment:
use order and limit instead of last and reverse

Reply:
done!